modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-azure_{context}"]
+= Mounting secrets from an external secrets store to a CSI volume
+
+TODO: intro
+
+This procedure assumes that you are using Azure Key Vault as the secrets store provider.
+
+.Prerequisites
+
+* You have installed the {secrets-store-operator}. See _Installing the Secrets Store CSI Driver_ for instructions.
+* You have installed a secrets store provider. See the appropriate vendor documentation for instructions. This procedure assumes that you have installed the link:https://azure.github.io/secrets-store-csi-driver-provider-azure/docs/getting-started/installation/[Azure Keyvault Provider].
+// TODO: Is there only a portion of these docs that they should follow? And any additional steps they need to take for OCP?
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a secret provider class to define your secrets store provider.
+
+.. Create a YAML file that defines the `SecretProviderClass` object:
++
+.Example `secret-provider-class.yaml`
+[source,yaml]
+----
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: my-azure-provider                 <1>
+spec:
+  provider: azure                         <2>
+  parameters:                             <3>
+    keyvaultName: "kvname"
+    objects: |
+      array:
+        - |
+          objectName: tls-cert
+          objectType: secret
+    tenantId: "tid"
+----
+<1> Specify the name for the provider secret provider class.
+<2> Specify the provider type.
+// TODO: Decide if we want to call out which are "supported"
+<3> Specify the provider-specific configuration parameters.
+
+.. Create the `SecretProviderClass` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f secret-provider-class.yaml
+----
+
+. Create a pod to use this secret provider class.
+
+.. Create a YAML file that defines the `Pod` object:
++
+.Example `pod.yaml`
+[source,yaml]
+----
+TODO: my-pod
```

Comment:
@ropatil010 @knelasevero

Do you have an example pod config that we can show here?

Reply:
Adding tc link [OCP-65181 ](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-65181:~:text=//%20keyvault%2Dsecrets.yaml) for tested secretproviderclass and dep yaml files in polarion. Can you PTAL once?